apps/site/assets/ts/schedule/components/ScheduleFinder.tsx

```diff
@@ -36,9 +41,31 @@ const ScheduleFinder = ({
   const [selectedDirection, setSelectedDirection] = useState(directionId);
   const [selectedOrigin, setSelectedOrigin] = useState<SelectedOrigin>(null);
 
+  const [query] = useQueryParams({
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    direction_id: StringParam,
```

Comment:
@alicia-mbta as we discussed in chat the other day, please adjust this `direction_id` as well as that in `apps/site/assets/ts/schedule/components/direction/reducer.ts`so that they're scoped and don't collide. That is, we should call the param here `schedule_finder[direction_id]` and the other `schedule_direction[direction_id]`.

Then, for consistency, we should also rename the `origin` param here to `schedule_finder[origin]`, and likewise `variant` in the other file to `schedule_direction[variant]`.

And of course the corresponding backend code should get adjusted to match.